localdev/Makefile

```diff
@@ -1,7 +1,7 @@
 down:
-	docker-compose down
+	docker compose down
 
 up: down
-	docker-compose build
-	docker-compose up --remove-orphans
+	docker compose build
+	docker compose up --remove-orphans
```

Comment:
I think it would be good to have a troubleshoot section that calls out that you need to update docker if you get
```bash
docker compose
docker: 'compose' is not a docker command.
See 'docker --help'
```
Also add min requirements for docker desktop (or generally say to update)